### DISCUSSION TITLE
Paste Table shout out

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#868] [Show and tell]
Table component is awesome, new icons are awesome, and Paste just makes Console so much more accessible :) The changes don't look like too much but with the new Paste Table we're now able to share this component with other teams, in internal and external tools. Big shout out to the team! 

Before:

![image](https://user-images.githubusercontent.com/6936736/96507463-0bb22b00-120e-11eb-8d5e-638385201bd1.png)

After:

![image](https://user-images.githubusercontent.com/6936736/96507289-ce4d9d80-120d-11eb-9216-4515375311a9.png)

**Comment by SiTaggart** [comment]
Nice, love this kind of content! The tables are starting to look much nicer, and being able to share them between teams is a huge win.

Once we get to start looking at filtering, we can definitely start making even more improvements to the experience.

**Comment by serifluous** [comment]
Love it!! 💛💛💛

Thanks so much for sharing.
